cves/CVE-2015-6826.yml

```diff
@@ -144,10 +166,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: No unit or regression tests were found.
```

Comment:
Something useful to verify:
https://github.com/FFmpeg/FFmpeg/tree/3197c0aa87a3b7190e17d49e6fbc7b554e4b3f0a/tests

No updates to unit tests (under `./tests/` in >= 6 years! A quick parusal of adjacent files shows no related tests for `rv34*`. In other words, good entries here.

Reply:
Yeah, I was pretty surprised by the lack of unit testing, especially for such a ubiquitous utility like FFmpeg.